jina/peapods/pods/k8slib/kubernetes_tools.py

```diff
@@ -104,7 +109,9 @@ def create(template: str, params: Dict, custom_resource_dir: Optional[str] = Non
             utils.create_from_yaml(__k8s_clients.k8s_client, path)
         except FailToCreateError as e:
             if e.api_exceptions[0].status == 409:
-                print('exists already')
+                    logger.info(
+        f'🔁\tThe resource already exists in the kubernetes cluster'
```

Comment:
Its already better than before, but maybe while you are on this anyway: We could include resource name and type here?

Reply:
Yeah, it will be better. But I am not sure if we can do it. The resource is converted to YAML, and we create the resource from the YAML directly. 

I am not familiar with Kubernetes client python library, will try to have the name and resource type in the log